ganga/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -751,12 +751,15 @@ def getOutputSandbox(self, outputDir=None, unpack=True):
 
         return True
 
-    def removeOutputData(self):
+    def removeOutputData(self, fileType = 'DiracFile'):
         """
         Remove all the LFNs associated with this job.
         """
         # Note when the API can accept a list for removeFile I will change
         # this.
+        if not fileType=='DiracFile':
+            return False
+
```

Comment:
Should this somehow check if inheriting from `DiracFile` instead to allow for somebody to write a different type of `DiracFile`.

Reply:
It is a good idea but I am not sure how it can be practically done as we only know the name of the file type - I am not sure that will tell you about inheritance.

Reply:
You are right. It is never a good idea to pass around strings for processing but that is not a failure of this piece of code.